ajax/libs/echarts-gl/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "echarts-gl",
+  "description": "Extension pack of ECharts providing 3D plots and globe visualization",
+  "version": "1.1.2",
```

Comment:
```suggestion
```
Please remove the `version` attr, the auto-update bot will populate this once it pulls in versioned files for the library.